common/src/main/java/com/wynntils/features/user/map/GuildMapFeature.java

```diff
@@ -4,15 +4,38 @@
  */
 package com.wynntils.features.user.map;
 
+import com.wynntils.core.config.Config;
 import com.wynntils.core.features.UserFeature;
 import com.wynntils.core.features.properties.FeatureCategory;
 import com.wynntils.core.features.properties.FeatureInfo;
+import com.wynntils.core.features.properties.RegisterKeyBind;
+import com.wynntils.core.keybinds.KeyBind;
 import com.wynntils.core.managers.Model;
+import com.wynntils.gui.screens.maps.GuildMapScreen;
+import com.wynntils.mc.utils.McUtils;
 import com.wynntils.wynn.model.territory.GuildTerritoryModel;
 import java.util.List;
+import org.lwjgl.glfw.GLFW;
 
 @FeatureInfo(category = FeatureCategory.MAP)
 public class GuildMapFeature extends UserFeature {
+    public static GuildMapFeature INSTANCE;
+
+    @Config
+    public boolean renderUsingLinear = true;
```

Comment:
This is more of a general, across all map feature config

Reply:
I didnt wanna use the other feature's config?